pkg/validation/validation_test.go

```diff
@@ -68,7 +68,7 @@ func TestMachineConfigPoolDuplicates(t *testing.T) {
 
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
-			err := MachineConfigPoolDuplicates(tc.trees)
+			err := MCPsDuplicates(tc.trees)
```

Comment:
MCPsDuplicates is not defined in this commit, I guess some rebase/devision issue happened